release-notes/unreleased/2096.upgrade.rst

```diff
@@ -0,0 +1,2 @@
+The minimal ``qiskit`` required version is now 1.3.  
+The QPY payload is now generated based on ``qiskit.qpy.QPY_VERSION`` and capped at QPY 13, which is supported by Qiskit 1.3.
```

Comment:
I'm not sure if this warrants a user note. Mostly the QPY format version used on the wire should be transparent to users and I'm not sure it's worth documenting

Reply:
my point was to alert about the bump in the dependency requirement. Maybe I'm explaining too much why. Shall I just:
 ```suggestion
```
?

Reply:
I would link it a bit more in the text than to explain the relationship.

```suggestion
Qiskit 1.3 introduces a new QPY version 13. This new format version offers better features and the
minimum required version of Qiskit was raised to start leveraging it for serializing circuits in job
submissions.
```